lib/logstash/inputs/http.rb

```diff
@@ -72,7 +80,7 @@ def run(queue)
         REJECTED_HEADERS.each {|k| req.delete(k) }
         req = lowercase_keys(req)
         body = req.delete("rack.input")
-        @codec.decode(body.read) do |event|
+        @codecs[req["content_type"]].decode(body.read) do |event|
```

Comment:
I think HTTP send a `content_type` everytime.. But I am not sure at 100%. 
Can this value be nil? Should we use `.fetch("content_type", "text/plain")` to guard agains't this possibility?